include/experimental/fundamental/v3/std/std_pair.hpp

```diff
@@ -41,6 +41,10 @@ inline namespace fundamental_v3
 
 #if __cplusplus >= 201402L || defined JASEL_DOXYGEN_INVOKED
 
+  namespace functor {
```

Comment:
This is not correct. 
pair can be a functor only if T1,T2 are the same type, isn't it?
_t is something like auto and catch any type.

Please, extract this to another issue so that we can manage it separately, independently of the warnings